layouts/_default/list.html

```diff
@@ -27,6 +27,7 @@
 			<h1>{{ .Title }}</h1>
 			<p class="lead">{{ .Params.lead | safeHTML }}</p>
 			{{ .Content }}
+			{{ partial "main/pages.html" . }}
```

Comment:
This adds a table of contents list to pages like https://docs.blubracket.com/how-to/ and https://docs.blubracket.com/faq/